src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+// @import "../utils/functions";
+// @import "../utils/mixins";
+// @import "../utils/extends";
+@import "../utils/variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  .logo {
```

Comment:
Is it header part? Or it's the separate BEM block?
Use scss nesting or move it to separate scss file